src/api/crypto/blockchain/Imp_blockchain.cpp

```diff
@@ -322,17 +328,39 @@ auto BlockchainImp::LoadTransactionBitcoin(const Txid& txid) const noexcept
 
 auto BlockchainImp::load_transaction(const Lock& lock, const TxidHex& txid)
     const noexcept
-    -> std::unique_ptr<opentxs::blockchain::block::bitcoin::Transaction>
+    -> std::unique_ptr<opentxs::blockchain::bitcoin::block::Transaction>
 {
-    return load_transaction(lock, api_.Factory().DataFromHex(txid));
+    auto proto = proto::BlockchainTransaction{};
+
+    return load_transaction(lock, txid, proto);
+}
+
+auto BlockchainImp::load_transaction(
```

Comment:
not better one method with out parameter with default values?

Reply:
MT-80